tsconfig.json

```diff
@@ -1,107 +1,107 @@
 {
   "compilerOptions": {
-    /* Visit https://aka.ms/tsconfig to read more about this file */
+    // Visit https://aka.ms/tsconfig to read more about this file 
```

Comment:
The changes in this file appear to be just formatting. Would it be worth adopting prettier in this repository in a subsequent PR?

Reply:
This change was a requirement of running `web-ext lint`, so we should probably adopt that rather than prettier, but good idea